cg/services/orders/order_service/order_service.py

```diff
@@ -40,3 +40,9 @@ def create_order(self, order_data: OrderIn) -> OrderResponse:
 
     def set_delivery(self, order_id: int, delivered: bool) -> None:
         self.store.update_order_delivery(order_id=order_id, delivered=delivered)
+
+    def update_delivered(self, order_id: int, delivered_analyses: int) -> None:
+        order: Order = self.store.get_order_by_id(order_id)
+        case_count: int = len(order.cases)
+        if delivered_analyses >= case_count:
```

Comment:
should the `>` here be able to happen? 
Is it possible to abstract away the check into function. The step of checking the number of delivered analyses to the case_count in the order is a bit difficult to follow.
perhaps:

```suggestion
        if analyses_for_cases_delivered():
```
I think either way works though.

Reply:
I think for microsalt you can start a partial analysis when not all samples are ready. I guess that could result in a larger than? And also I guess hidden analyses could potentially be marked as delivered..

Reply:
Also, maybe the uncertainties can be cleared with a short docstring?

Reply:
Yes, a docstring can also make it clearer!

Reply:
Resolved in https://github.com/Clinical-Genomics/cg/pull/3097/commits/4b85a65372a4217ded9f66f2db69ef8de94da352